gen/schema_gen.go

```diff
@@ -314,7 +314,7 @@ func (g *schemaGen) generate2(name string, schema *jsonschema.Schema) (ret *ir.T
 			}
 			if t.Validators.Int.Set() {
 				switch t.Primitive {
-				case ir.String, ir.ByteSlice:
+				case ir.Int, ir.Int8, ir.Int16, ir.Int32, ir.Int64, ir.ByteSlice:
```

Comment:
```suggestion
				case ir.Int,
					ir.Int8,
					ir.Int16,
					ir.Int32,
					ir.Int64,
					ir.Uint,
					ir.Uint8,
					ir.Uint16,
					ir.Uint32,
					ir.Uint64:
```

Reply:
Fixed in https://github.com/ogen-go/ogen/pull/735/commits/dc1c5ea0bda977e6001a117e5bfb7e3b72e09d2c.